src/components/builder/loader.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import {Meta, StoryObj} from '@storybook/react';
+
+import Loader from '@/components/builder/loader';
+
+export default {
+  title: 'Formio/Builder/Loader',
+  component: Loader,
+  parameters: {
+    controls: {hideNoControlsWarning: true},
+    docs: {
+      source: {
+        type: 'dynamic',
+        excludeDecorators: true,
+      },
+    },
```

Comment:
I don't think this is needed anymore, it was a Storybook 6/7 thing when using MDX for documentation.

Reply:
Check, it indeed wasn't needed anymore.. Ive removed it